_pages/how-we-work/research-guidelines.md

```diff
@@ -58,19 +56,18 @@ As researchers, however, we need to follow the principle of beneficence. So just
 
 ### Storing PII and sharing research data
 
-The best place to store PII is not to store it at all. The best place online, if you _must_ collect it, is in a locked-down document on Google Drive that is only accessible to the immediate research team. Using that document, link each participant to a pseudonym form (eg a codename or number such "PO1") Research reports and notes should contain only those pseudonyms. When it comes time to release reports outside of the immediate research team, refer back to your locked-down file for the correct sharing permissions. Our consent forms by default ask participants to approve sharing quotations with the public, but it's always possible for someone to edit or adapt the forms to give more options. If you have any paper documents with PII, they always go into a locked file cabinet. 
+The best place to store PII is not to store it at all. The best place online, if you _must_ collect it, is in a locked-down document on Google Drive that is only accessible to the immediate research team. Using that document, link each participant to a pseudonym form (eg a codename or number such "PO1") Research reports and notes should contain only those pseudonyms. When it comes time to release reports outside of the immediate research team, refer back to your locked-down file for the correct sharing permissions. If you have any paper documents with PII, they always go into a locked file cabinet. 
 
 A good rule is to check with your project lead before sharing information outside the immediate team. What has been approved by the GSA may not be approved by partners. Check the Appendix in [Design page](/design/) for detailed information about who can see what, when. The social media, collaboration, and security classes in GSA's Online University can also be helpful in managing access. 
```

Comment:
Minor stylistic typo from the previous version I just noticed: I don't think we use `the GSA`. The [content guide](https://content-guide.18f.gov/abbreviations-and-acronyms/) isn't clear, but I believe it should be just `GSA`, with no definite article.